### PR TITLE
Rename transfer call to registerPayment

### PR DESCRIPTION
**Pull request by CarrilloJuan**

### CODE REVIEW

File: buyer-api/src/services/signingService.js [hunk 1]
```diff
@@ -38,6 +38,6 @@ const createSigningMethod = endpoint => payload =>
 
 export const signCreateDataOrder = createSigningMethod('/sign/create-data-order');
 export const signCloseDataOrder = createSigningMethod('/sign/close-data-order');
-export const signTransfer = createSigningMethod('/sign/bat-pay/register-payment');
+export const signregisterPayment = createSigningMethod('/sign/bat-pay/register-payment');
```
Comment (nicoayala) [inline buyer-api/src/services/signingService.js]:
```suggestion
export const signRegisterPayment = createSigningMethod('/sign/bat-pay/register-payment');
```

File: buyer-api/src/services/notaryService.js [hunk 2]
```diff
@@ -43,25 +42,22 @@ export const transferNotarizationResult = async (notarizationRequestId) => {
...
     metadata: transactionHash,
     fee,
-    newCount: '0x',
-    rootHash: '0x',
+    newCount: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    rootHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
   };
 
-  await addTransactionJob('Transfer', payload);
+  await addTransactionJob('registerPayment', payload);
```
Comment (nicoayala) [inline buyer-api/src/services/notaryService.js]:
```suggestion
  await addTransactionJob('RegisterPayment', payload);
```

File: buyer-api/src/routes/notarizationResult.js [hunk 3]
```diff
@@ -57,9 +57,9 @@ const router = Router();
  *       payDataHash:
  *         type: string
  *         description: Seller ids array hash.
- *       lock:
+ *       lockingKeyHash:
  *         type: string
- *         description: The lock for the BatPay.Transfer.
+ *         description: The lockingKeyHash for the BatPay.Transfer.
```
Comment (nicoayala) [inline buyer-api/src/routes/notarizationResult.js]:
```suggestion
 *         description: The lockingKeyHash for the BatPay.registerPayment.
```

File: buyer-api/src/services/notaryService.js [hunk 4]
```diff
@@ -43,25 +42,22 @@ export const transferNotarizationResult = async (notarizationRequestId) => {
...
-      sellers,
-      lock,
+      notarizationFee: fee, orderId, sellers, lockingKeyHash,
     },
   } = await notarizations.fetch(notarizationRequestId);
   const { transactionHash, price } = await dataOrders.fetchByDxId(orderId);
 
   const payload = {
-    amount: fromWib(price),
+    amount: Number(fromWib(price)),
```
Comment (Meketh) [inline buyer-api/src/services/notaryService.js]: This cast can cause overflow, since `fromWib` returns a `BigNumber`.

**Pull request merged**